cors/resources/cors-makeheader.py

```diff
@@ -1,67 +1,67 @@
 import json
 
 def main(request, response):
-    origin = request.GET.first("origin", request.headers.get('origin'))
+    origin = request.GET.first(b"origin", request.headers.get(b'origin'))
 
-    if "check" in request.GET:
-        token = request.GET.first("token")
+    if b"check" in request.GET:
+        token = request.GET.first(b"token")
         value = request.server.stash.take(token)
         if value is not None:
-            if request.GET.first("check", None) == "keep":
+            if request.GET.first(b"check", None) == b"keep":
                 request.server.stash.put(token, value)
-            body = "1"
+            body = u"1"
         else:
-            body = "0"
-        return [("Content-Type", "text/plain")], body
+            body = u"0"
+        return [(b"Content-Type", b"text/plain")], body
 
 
-    if origin != 'none':
-        response.headers.set("Access-Control-Allow-Origin", origin)
-    if 'origin2' in request.GET:
-        response.headers.append("Access-Control-Allow-Origin", request.GET.first('origin2'))
+    if origin != None:
```

Comment:
This seems wrong. Now if origin is "none" it would get set.

Reply:
Well spotted! I couldn't remember why I replaced this line. Code updated. Thank you!

Reply:
Well, it should probably become `b"none"` or some such, right?

Reply:
Yes :).